gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/rest/api/service/GroupService_UpdateTest.java

```diff
@@ -76,7 +76,10 @@ public void shouldUpdateGroup() throws Exception {
         updatedGroupEntity.setRoles(Maps.<RoleScope, String>builder().put(RoleScope.API, "OWNER").build());
         updatedGroupEntity.setSystemInvitation(false);
 
-        when(groupRepository.findById(GROUP_ID)).thenReturn(Optional.of(Mockito.mock(Group.class)));
+        final Group groupMock = mock(Group.class);
+        when(groupMock.getApiPrimaryOwner()).thenReturn("api-primary-owner");
```

Comment:
Using a mock here feels a bit strange to me, why not use a Group instance directly instead?

 ```suggestion
        final Group group = new Group();
        group.setApiPrimaryOwner("api-primary-owner");
```

Reply:
yes, it was returning a mock, so I kept it, I will change that :)